moto/ec2/models.py

```diff
@@ -2019,13 +2019,13 @@ def __init__(
         to_port,
         ip_ranges,
         source_groups,
-        prefix_list_ids=None,
+        prefix_list_ids=[],
```

Comment:
Default arguments should always be None, IMO, as having a list as default can lead to really tricky bugs.
(https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments if you want a clear/concise summary of what can go wrong)